packages/python/devel/pathlib/package.mk

```diff
@@ -22,7 +22,7 @@ PKG_SHA256="6940718dfc3eff4258203ad5021090933e5c04707d5ca8cc9e73c94a7894ea9f"
 PKG_ARCH="any"
 PKG_LICENSE="MIT"
 PKG_SITE="http://pathlib.readthedocs.org"
-PKG_URL="http://pypi.python.org/packages/source/p/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.gz"
+PKG_URL="https://pypi.python.org/packages/ac/aa/9b065a76b9af472437a0059f77e8f962fe350438b927cb80184c32f075eb/pathlib-1.0.1.tar.gz"
```

Comment:
Thank you. Please change just change it to https:
https://pypi.python.org/packages/source/p/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.gz
https://pypi.python.org/packages/source/p/pathlib/pathlib-1.0.1.tar.gz is working for me

Reply:
https://bitbucket.org/pitrou/pathlib/downloads/?tab=tags
is also a dl fallback :)